lib/adapters/http.js

```diff
@@ -171,8 +171,12 @@ module.exports = function httpAdapter(config) {
       transport = isHttpsProxy ? httpsFollow : httpFollow;
     }
 
-    if (config.maxContentLength && config.maxContentLength > -1) {
-      options.maxBodyLength = config.maxContentLength;
+    if (config.maxBodyLength > -1) {
+      options.maxBodyLength = config.maxBodyLength;
+      if (data.length > config.maxBodyLength) {
+        reject(createError('maxBodyLength size of ' + config.maxBodyLength + ' exceeded',
+          config, null, null));
+      }
```

Comment:
I suggest to remove this check. Because `follow-redirects` handles it. Or `options.maxBodyLength` do not need to be passed.

Reply:
@RubenVerborgh Can you help review for this pull request? Thanks.

Reply:
@chinesedfan I added it in for the sake of consistency with the `maxContentLength` option. 
https://github.com/axios/axios/blob/master/lib/adapters/http.js#L217

However, that one is not handled by `follow-redirrcts` so I'm OK with getting rid of the above condition.

Reply:
Check removed